test/test_importer.py

```diff
@@ -1672,6 +1672,12 @@ def test_reimported_item_preserves_art(self):
         if new_artpath != old_artpath:
             self.assertNotExists(old_artpath)
 
+    def test_reimported_album_has_new_flexattr(self):
+        self._setup_session()
+        self.assertIsNone(self._album().get("bandcamp_album_id"))
+        self.importer.run()
+        self.assertEqual(self._album().bandcamp_album_id, "bc_url")
```

Comment:
I confirmed this fails without my change :)

Reply:
Thanks for adding the test!

I'd suggest rebasing this PR off `master` - you will find that all unittest assertions like `assertIsNone` and `assertEqual` have been replaced by `assert` statements.